Makefile

```diff
@@ -199,6 +202,14 @@ golangci-lint/%: $(GOLANGCI_LINT)
 		&& cd $(DIR) \
 		&& $(GOLANGCI_LINT) run --allow-serial-runners $(ARGS)
 
+.PHONY: testifylint
+testifylint: $(OTEL_GO_MOD_DIRS:%=testifylint/%)
```

Comment:
Why do we need testifylint as a standalone command? We already have it as part of golangci-lint.

Reply:
Can't you install it locally, without the makefile changes to be able to run it?

Reply:
You could also make the makefile changes locally, but not commit them.

Reply:
That doesn't seem so tedious:

```
git add . && git reset Makefile
```

Reply:
AFAIK `golangci-lint --fix` still returns an error if an issue is not fixed